test/save_test.rb

```diff
@@ -41,6 +41,15 @@ def test_update
     @record.save
   end
 
+  def test_update_no_change
+    # Regular flow, write index id, write index id/tile, delete data blob since Record has changed
+    expect_cache_delete("#{NAMESPACE}attr:Item:id:id/title:#{cache_hash('"1"/"bob"')}").never
+    expect_cache_delete("#{NAMESPACE}attr:Item:id:title:#{cache_hash('"bob"')}").never
+    expect_cache_delete(@blob_key).never
+
+    @record.save
```

Comment:
If the return value for the `save` isn't going to be checked, then use `save!` instead to make sure the test fails in an easy to debug way if there are unexpected validation errors.  You can also make this change when refactoring existing tests that have this anti-pattern.